src/js/modules/product-subscription-abtest.js

```diff
@@ -0,0 +1,16 @@
+$('#subscriptionCheckbox').on('change', function () {
+	const subscriptionVariant = $('.product-variant--subscription');
```

Comment:
1vars used from other functions must be declared outside of the function so we don't need to write them again.